netlify/functions/tweets.cjs

```diff
@@ -0,0 +1,29 @@
+const Twitter = require('twitter')
+require('dotenv').config()
+
+const client = new Twitter({
+  consumer_key: process.env.TWITTER_CONSUMER_KEY,
+  consumer_secret: process.env.TWITTER_CONSUMER_SECRET,
+  bearer_token: process.env.TWITTER_BEARER_TOKEN,
+})
+
+const params = {
+  screen_name: 'AiryxOS',
+  count: 10,
+  exclude_replies: true,
+  include_rts: false,
+  tweet_mode: 'extended',
+}
+
+exports.handler = async function () {
+  try {
+    const timeline = await client.get('statuses/user_timeline', params)
```

Comment:
**Caveat:** Serverless functions by nature are stateless so expect more requests to the Twitter API than before.

Reply:
Understood! Should be OK for now - our limit on the API is reasonable and we have caching through the CDN.